src/js/edu-benefits/1995/config/form.js

```diff
@@ -138,25 +138,12 @@ const formConfig = {
               'ui:title': 'Do you have any new periods of service to record since you last applied for education benefits?',
               'ui:widget': 'yesNo'
             },
-            toursOfDuty: {
-              'ui:title': 'Service periods',
-              'ui:options': {
-                itemName: 'Service Period',
-                viewField: ServicePeriodView,
-                hideTitle: true,
-                expandUnder: 'view:newService'
-              },
-              items: {
-                serviceBranch: {
-                  'ui:title': 'Branch of service'
-                },
-                dateRange: dateRange.uiSchema(
-                  'Start of service period',
-                  'End of service period',
-                  'End of service must be after start of service'
-                )
-              }
-            }
+            toursOfDuty: (() => {
```

Comment:
Aha! I was looking for that, but only came up with `_.set()` and `_.assign()`, and they both just clobbered what was already in there. `_.merge()` it is!